src/sentry/api/serializers/models/organization.py

```diff
@@ -87,6 +87,9 @@ def serialize(self, obj, attrs, user):
 
         for project in project_list:
             project._organization_cache = obj
+            project.platforms = ProjectPlatform.objects.filter(
```

Comment:
We can just fetch all of these at once by doing:

```python
platform_list = list(ProjectPlatform.objects.filter(project_id__in=[p.id for p in project_list]).values_list('project_id', 'platform'))

platforms_by_project_id = defaultdict(list)
for project_id, platform in platform_list:
    platforms_by_project_id[project_id].append(platform)

for project in project_list:
    project._organization_cache = obj
    project.platforms = platforms_by_project_id[project.id]
```

Lastly, please don't use `project.platforms` as a way to store this data. Since this instance is a model, it's confusing if you're just tacking on random properties. And it'll be more problematic if we were to say, add a field called `platforms`. (There's already one called `platform`.